server/log.go

```diff
@@ -15,9 +15,9 @@ var trace int32
 var debug int32
 
 var log = struct {
-	sync.Mutex
+	*sync.Mutex
```

Comment:
Unit test is storing the value of `log` before the test and replacing it after to prevent side effects, so govet complains because it's copying the mutex value (motivation for #501).